Sources/TripKit/model/TKBookingTypes.swift

```diff
@@ -102,9 +103,10 @@ public enum TKBooking {
       case productType
       case explicitValidity = "valid"
       case validFor
-      case validFrom
+      case validFrom = "validFromTimestamp"
```

Comment:
Does this depend on a backend change or was it just wrong?

Reply:
It'd done because of a BE change.